lib/install-php-ubuntu.sh

```diff
@@ -5,17 +5,22 @@ set -eo pipefail
 release=$(lsb_release -cs)
 version=$1
 
+# see https://www.cyberciti.biz/faq/how-to-speed-up-apt-get-apt-command-ubuntu-linux/
+sudo apt-get install software-properties-common
```

Comment:
this might not be necessary, as it seems the package is alerady installed in the used operation system:

![grafik](https://user-images.githubusercontent.com/120441/65435517-ec857600-de20-11e9-805a-1e9254eb01a2.png)

Reply:
Thx!
Removed